rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/OneScanCovidRDTBarcodeFactoryTest.java

```diff
@@ -97,6 +101,9 @@ public void testOnActivityResultShouldPopulateCorrectData() throws Exception {
         });
         RDTJsonFormUtilsShadow.setJsonObject(deviceDetailsWidget);
 
+        Barcode barcode = new Barcode();
+        barcode.displayValue = barcodeVals;
+        intent.putExtra(JsonFormConstants.BARCODE_CONSTANTS.BARCODE_KEY, barcode);
```

Comment:
This is a duplicate of https://github.com/onaio/rdt-standard/pull/656/files#diff-7c8f82faba856576126f062799a7e505ed0666e6f93cdd2541eec5e2733dd4f0R132. I think it could move to a private method.

Reply:
fixed.